05_DeleteUser.php

```diff
@@ -77,7 +77,7 @@
 					<form method = "post" id= "06_delete_user.php">
 						<h6><label for = 'login'>User Name:</label>
 							<input type = "text" name = "UserName" placeholder="Enter user name"/></h6>
-						<h6><input type = "submit" value = "Delete" /></h6><br>
+						<h6><input class="w3-padding-small w3-button w3-red" type = "submit" value = "Delete" /></h6><br>
```

Comment:
made button red as this represents delete and makes sense than using green; Graeme will see the difference between the pages easier